tests/dummy/app/components/code-snippet/component.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import { getCodeSnippet } from 'ember-code-snippet';
+import { computed } from '@ember/object';
+
+export default class CodeTemplateToggleComponent extends Component {
+  tagName = '';
+
+  @computed('name')
+  get snippet() {
+    return getCodeSnippet(this.name);
+  }
+
+  get language() {
+    if (this.snippet.language === 'handlebars') {
```

Comment:
ember-prism expects hbs files to use the language "markup".  This feels a bit hacky. Would love a better solution